sormas-ui/src/main/java/de/symeda/sormas/ui/events/importer/LineListingImportLayout.java

```diff
@@ -0,0 +1,51 @@
+package de.symeda.sormas.ui.events.importer;
+
+import java.io.IOException;
+import java.util.function.Consumer;
+
+import com.opencsv.exceptions.CsvValidationException;
+import com.vaadin.server.ClassResource;
+import com.vaadin.server.Page;
+import com.vaadin.server.StreamResource;
+import com.vaadin.ui.Notification;
+import com.vaadin.ui.Notification.Type;
+
+import de.symeda.sormas.api.FacadeProvider;
+import de.symeda.sormas.api.i18n.I18nProperties;
+import de.symeda.sormas.api.i18n.Strings;
+import de.symeda.sormas.ui.importer.AbstractImportLayout;
+import de.symeda.sormas.ui.importer.ImportReceiver;
+
+public class LineListingImportLayout extends AbstractImportLayout {
```

Comment:
I don't think you need to copy this class here; there's no line listing for events